cmd/nebula-cert/print.go

```diff
@@ -49,20 +49,19 @@ func printCert(args []string, out io.Writer, errOut io.Writer) error {
 	var qrBytes []byte
 	part := 0
 
+	var jsonCerts []cert.Certificate
```

Comment:
Main thing to note here is that we don't have a human string version anymore. Its always json. Do we want this?